packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-point-settings.tsx

```diff
@@ -104,28 +105,39 @@ const LockedPointSettings = (props: Props) => {
 
     const isDefiningPoint = !onMove && !onRemove;
 
-    /**
-     * Get a prepopulated aria label for the point.
-     *
-     * If the point has no labels, the aria label will just be
-     * "Point at (x, y)".
-     *
-     * If the point has labels, the aria label will be
-     * "Point at (x, y) with label1, label2, label3".
-     */
-    function getPrepopulatedAriaLabel() {
-        let visiblelabel = "";
-        if (labels && labels.length > 0) {
-            visiblelabel += ` ${labels.map((l) => l.text).join(", ")}`;
-        }
-        let str = `Point${visiblelabel} at (${coord[0]}, ${coord[1]})`;
+    const [prepopulatedAriaLabel, setPrepopulatedAriaLabel] =
+        React.useState("");
+
+    React.useEffect(() => {
+        /**
+         * Get a prepopulated aria label for the point, with the math
+         * details converted into spoken words.
+         *
+         * If the point has no labels, the aria label will just be
+         * "Point at (x, y)".
+         *
+         * If the point has labels, the aria label will be
+         * "Point at (x, y) with label1, label2, label3".
```

Comment:
I think this comment should be something like

```suggestion
         * If the point has labels, the aria label will be
         * "Point label1, label2, label3 at (x, y)".
```

Reply:
Good catch!